### PR TITLE
Object/add-on .blank 

### PR DESCRIPTION
**Pull request by quintin-reineccius**
I messed up my branching

### CODE REVIEW

File: array/last.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+if (!Array.prototype.last) {
+  Object.defineProperty(Array.prototype, 'last', {
+    value: function(num) {
+      var n = num ? num : 1;
+      return this.splice(this.length - n, this.length);
```
Comment (dphaener) [inline array/last.js:5]: Couldn't this just be `this.splice(-1, n)`?

File: object/blank.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  Object.defineProperty(Object.prototype, "blank", {
+    get: function() {
+      var key = 0;
+      if (key < Object.keys(this).length){
+        return true;
+      }
+      else{
+        return false;
+      }
+    }
```
Comment (dphaener) [inline object/blank.js:5]: I think you could greatly simplify this:

``` js
return Object.keys(this).length === 0;
```

File: tests/object/blank.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+var chai = require('chai');
+var expect = chai.expect;
+
+require('../../object/blank');
+
+describe('blank', function() {
+
+  it('should return true if object has items', function() {
```
Comment (dphaener) [inline tests/object/blank.js]: Shouldn't blank return false if the object is NOT blank?

Comment (zackify) [inline tests/object/blank.js]: haha someone wasn't thinking @rampage1213

**Comment by quintin-reineccius** [comment]
I mixed that up. Should work right now

**Pull request closed without merging**